### PR TITLE
React enzyme tests

### PR DESCRIPTION
**Pull request by shilman**
**Don't merge**. This is the simplest possible unit test of TodoItem, just to see if I could get it working and it works. Most of the mocha/enzyme setup is borrowed from [react-cdk](https://github.com/kadirahq/react-cdk).

How to try it out:
```
# using a 4.0+ version of node
npm install
npm run test-enzyme 
```

The code is hideous, but I wanted to share in case it helps move things forward. Feedback wanted:
- systematic way to do the testdouble thing or import meteor packages?
- way to do this from within meteor test?
- better directory structure?

### CODE REVIEW

**Comment by shilman** [comment]
@tmeasday @ffxsam I realize this is not a great PR, or perhaps shouldn't even be a PR since it doesn't build, but any feedback or nudges in the right direction would be much appreciated.

**Comment by shilman** [comment]
@tmeasday @ffxsam I realize this is not a great PR, or perhaps shouldn't even be a PR since it doesn't build (though it should actually work when you run it on your machine in spite of the errors!). Nevertheless, any feedback or nudges in the right direction would be much appreciated.

File: tests/enzyme.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```
Comment (tmeasday) [inline tests/enzyme.js]: Is there a reason you can't put this line inside a regular Meteor/mocha test (run via `practicalmeteor:mocha` etc?)

**Comment by tmeasday** [comment]
@shilman sorry, only just had a chance to look at it. I'm wondering why we can run Mocha via the Meteor build tool + testing infrastructure? Is there something special about the enzyme import that would stop that happening?

**Comment by shilman** [comment]
@tmeasday per your encouragement, I gave it a try with the Meteor build tool + testing infrastructure and issued a new PR in #141, and indeed it's a lot cleaner. I'm closing this PR for now.

**Pull request closed without merging**